lib/assisted_workflow/addons/github.rb

```diff
@@ -22,12 +22,8 @@ def description
       @issue.body.to_s.gsub("\r\n", "\n")
     end
     
-    def other_id
```

Comment:
It was used [here](https://github.com/inaka/assisted_workflow/pull/4/files#diff-a9b3f7b05650ed3ad1382d1622534bf0R102) to name the branch, instead I use github username